### PR TITLE
README.md: Switch mirroring around

### PR DESCRIPTION
**Pull request by fabianfett**
This pr switches all links in the README around regarding gitlab and github being mirrored.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,20 +3,19 @@
...
     <a href="https://github.com/swift-server/sswg/blob/master/process/incubation.md#sandbox-level"><img src="https://img.shields.io/badge/sswg-sandbox-lightgrey.svg" alt="SSWG Maturity"></a>
-    <a href="https://gitlab.com/Mordil/RediStack/blob/master/LICENSE.txt"><img src="https://img.shields.io/badge/License-Apache%202.0-yellow.svg" alt="Apache 2 License"></a>
+    <a href="https://github.com/swift-server/RediStack/blob/main/LICENSE.txt"><img src="https://img.shields.io/badge/License-Apache%202.0-yellow.svg" alt="Apache 2 License"></a>
     <a href="https://swift.org"><img src="https://img.shields.io/badge/Swift-5.5+-orange.svg" alt="Swift 5.5+"></a>
     <a href="https://redis.io"><img src="https://img.shields.io/badge/Redis-5.0-red.svg" alt="Redis 5.0"></a>
 </p>
 
 <p>
-    <a href="https://gitlab.com/Mordil/RediStack/pipelines"><img src="https://gitlab.com/Mordil/RediStack/badges/master/pipeline.svg" alt="Build Status"></a>
     <a href="https://docs.redistack.info"><img src="https://docs.redistack.info/badge.svg" alt="Documentation Coverage"></a>
```
Comment (Joannis) [inline README.md:12]: I was about to make a PR for this as well. Can we remove the links to redistack.info? I don't think we have access to the domain.

Comment (Joannis) [reply README.md:12]: The codeclimate.io link should also go

**Review by Joannis — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (fabianfett) [reply README.md:12]: Codeclimate is #55.

**Review by Joannis — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -76,29 +71,27 @@ print(result) // Optional("some value")
...
 ## Documentation
 
-The docs for the latest tagged release are always available at [docs.redistack.info](http://docs.redistack.info).
+The docs for the latest tagged release are always available at the [Swift Package Index][Documentation].
 
 ## Questions
 
-For bugs or feature requests, file a new [issue](https://gitlab.com/mordil/RediStack/-/issues).
-
-For all other support requests, please email [support@redistack.info](mailto:support@redistack.info).
```
Comment (Joannis) [inline README.md:85]: I agree with removing this support link, but we might benefit from pointing towards the Swift OpenSource slack.

File: README.md [hunk 3]
```diff
@@ -101,12 +90,19 @@ This policy is to balance the desire for as much backwards compatibility as poss
 
 ## License
 
-[Apache 2.0](https://gitlab.com/Mordil/RediStack/blob/master/LICENSE.txt)
+[Apache 2.0][Apache License]
```
Comment (Joannis) [inline README.md:93]: Should we link to the local `LICENSE.md`?

Comment (fabianfett) [reply README.md:93]: See at the bottom, where the links are actually located.

File: README.md (hunk 2 above)
Comment (fabianfett) [reply README.md:70]: Let's do this as a follow up?

**Review by Joannis — APPROVED** [review]

**Pull request merged**